### PR TITLE
Connatix bugfixing

### PR DESCRIPTION
**Pull request by cristisilav**
- fix mute/unmute controls in the docking interface
- send dock/undock events to the iframe

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cristisilav im trying to reproduce the bug in the examples. would you mind giving me a step by step guide on how to reproduce the bug.

I'm a little hesitant if the mutation observer approach here is the way to go do observe docking. trying to see if there is a low level event that we can programmatically listen to instead

cc @powerivq

**Comment by erwinmombay** [comment]
also a heads up that we are currently in code freeze and will exit out of the freeze first week of January

**Comment by cristisilav** [comment]
> @cristisilav im trying to reproduce the bug in the examples. would you mind giving me a step by step guide on how to reproduce the bug.
> 
> I'm a little hesitant if the mutation observer approach here is the way to go do observe docking. trying to see if there is a low level event that we can programmatically listen to instead
> 
> cc @powerivq

Regarding the bug, the issue was that the mute/unmute control in the docking interface would not always reflect the actual volume of the player, which happened because I didn't explicitly emit that mutedOrUnmutedEvent.
As for docking observing, I couldn't find any event that was emitted (outside of obvious options such as a transitionstart or animationstart). By any chance have you found any other reliable, docking-specific events? Thank you!

File: extensions/amp-connatix-player/0.1/amp-connatix-player.js [hunk 1]
```diff
@@ -323,8 +330,33 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
...
+          this.sendCommand_(
+            mutation.target.classList.contains('i-amphtml-video-docked')
+              ? 'dock'
+              : 'undock'
+          );
+        }
+      }
+    };
+
+    const observer = new MutationObserver(callback);
```
Comment (powerivq) [inline extensions/amp-connatix-player/0.1/amp-connatix-player.js:354]: Can you move the observer to a member property. When the player is being destroyed, we can clean up the observer.

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by erwinmombay** [comment]
@cristisilav I talked with @powerivq and we couldn't find a low level event to listen to, so lets go ahead and proceed with this change with the minor request that @powerivq asked for and we can merge this

**Review by erwinmombay — APPROVED** [review]

**Comment by erwinmombay** [comment]
@cristisilav could you rebase from main to fetch the latest updates. i think that should fix the circle ci/test failure

**Review by powerivq — APPROVED** [review]

**Pull request merged**